nixos/modules/services/web-apps/grocy.nix

```diff
@@ -24,6 +24,19 @@ in {
       '';
     };
 
+    nginx.securityHeaders = mkOption {
+      type = types.str;
```

Comment:
This should be `types.lines`. Otherwise it's not possible to append config add several places.

Reply:
Changed in c84a3081f905e125cd765786d1c89b725b30554d.